pkg/metrics/metrics.go

```diff
@@ -20,15 +20,19 @@ import (
 	"fmt"
 	"net/http"
 	"os"
+	"strings"
 
 	"k8s.io/component-base/metrics"
 	"k8s.io/klog/v2"
+	"sigs.k8s.io/gcp-compute-persistent-disk-csi-driver/pkg/common"
 )
 
 const (
 	// envGKEPDCSIVersion is an environment variable set in the PDCSI controller manifest
 	// with the current version of the GKE component.
-	envGKEPDCSIVersion = "GKE_PDCSI_VERSION"
+	envGKEPDCSIVersion  = "GKE_PDCSI_VERSION"
+	hyperdiskDriverName = "hyperdisk.csi.storage.gke.io"
```

Comment:
I don't think we need to fake the driver name since we are distinguishing by disk type

Reply:
But if in the long run we want to make some changes in the sidecar to distinguish two different driver_name, this step seems a bit more proactive?